.dockerignore

```
@@ -0,0 +1,2 @@
+.github
```

```suggestion
.git
```

.git が肥大化するとDocker buildに時間がかかるようになるのでそれで追加するやつです。

---

ありがとうございます。
suggestion をそのまま反映させました！